controllers/token_scorer/token_scorer.py

```diff
@@ -51,15 +100,15 @@ def _record_log_entry(self, entry: ClaimLogEntry) -> None:
         self._log.append(entry)
         self._log_is_dirty = True
 
-    def log_token_value_change(
+    def log_location_change(
         self,
         token_code: TargetInfo,
-        token_value: int,
+        location: ScoringLocation,
         claim_time: float,
     ) -> None:
-        self._record_log_entry(ClaimLogEntry(token_code, token_value, claim_time))
+        self._record_log_entry(ClaimLogEntry(token_code, location, claim_time))
         print(  # noqa:T001
-            f"{token_code} NOW WORTH {token_value} TO {token_code.owner.name} "
+            f"{token_code} NOW AT {location.name} TO {token_code.owner.name} "
```

Comment:
"TO" is now possibly not the right preposition?

Reply:
Ah, I'd meant to fix that too. Not sure what the right spelling is though.

Reply:
84c9df7